editor/editor_node.cpp

```diff
@@ -6065,8 +6065,8 @@ EditorNode::EditorNode() {
 	EDITOR_DEF_RST("interface/scene_tabs/restore_scenes_on_load", true);
 	EDITOR_DEF_RST("interface/inspector/default_property_name_style", EditorPropertyNameProcessor::STYLE_CAPITALIZED);
 	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::INT, "interface/inspector/default_property_name_style", PROPERTY_HINT_ENUM, "Raw,Capitalized,Localized"));
-	EDITOR_DEF_RST("interface/inspector/default_float_step", 0.001);
-	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::FLOAT, "interface/inspector/default_float_step", PROPERTY_HINT_RANGE, "0,1,0"));
+	EDITOR_DEF_RST("interface/inspector/default_float_step_exponent", -3);
+	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::FLOAT, "interface/inspector/default_float_step_exponent", PROPERTY_HINT_RANGE, "-14,0,1"));
```

Comment:
-14 would be a really really tiny step value. It would basically disable the step, practically speaking. And it wouldn't be very useful with single-precision floats. But maybe it's good to allow this anyway.

Reply:
@aaronfranke  yeah, give the user enough options and let the user decide how to choose, I even thought about providing a bigger range containing some positive values (eg. [-14, 22]).

As for the name, I'm not good at naming names. But this property seems to be used only for step-related concepts, so containing `step` in the name might not be too much confused. `default_float_step_decimals` may be better.